jib-core/src/main/java/com/google/cloud/tools/jib/http/FailoverHttpClient.java

```diff
@@ -271,7 +308,7 @@ private Response call(String httpMethod, URL url, Request request, HttpTransport
     try {
       Response response = new Response(httpRequest.execute());
       synchronized (responsesCreated) {
-        responsesCreated.addLast(response);
+        responsesCreated.add(response);
```

Comment:
For consistency, I think we can make the same change with `trasnsportsCreated` on L323.